components/eam/src/physics/cam/aer_rad_props.F90

```diff
@@ -197,6 +213,12 @@ subroutine aer_rad_props_sw(list_idx, dt, state, pbuf,  nnite, idxnite, is_cmip6
    real(r8), pointer :: ext_cmip6_sw(:,:,:)
    real(r8) :: ext_cmip6_sw_inv_m(pcols,pver,nswbands)! short wave extinction in the units of 1/m
 
+   ! for Strat. AOD output
+   !if (is_output_interactive_volc) then 
+   !real(r8) :: ext_sao_sw(pcols,0:pver,nswbands) !
+   !real(r8) :: ssa_sao   (pcols,0:pver,nswbands) !
+   !real(r8) :: af_sao    (pcols,0:pver,nswbands) !
+   !endif
    !-----------------------------------------------------------------------------
```

Comment:
same thing here. Either remove it for cleaner merge or uncomment them.

Reply:
deleted it.